CONTRIBUTING.md

```diff
@@ -17,11 +17,11 @@ To set up your environment and begin working on our code, see [Developing for Te
   - [Adopting good development principles](https://github.com/tektoncd/community/blob/main/standards.md#principles)
   - [Writing useful commit messages](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)
 - [Contacting other contributors](https://github.com/tektoncd/community/blob/main/contact.md)
-- [Tekton development processes](https://github.com/tektoncd/community/blob/main/process.md), including:
-  - [Finding things to work on](https://github.com/tektoncd/community/blob/main/process.md#finding-something-to-work-on)
-  - [Proposing new features](https://github.com/tektoncd/community/blob/main/process.md#proposing-features)
-  - [Performing code reviews](https://github.com/tektoncd/community/blob/main/process.md#reviews)
-  - [Becoming a code owner](https://github.com/tektoncd/community/blob/main/process.md#owners)
+- [Tekton development processes](https://github.com/tektoncd/community/tree/main/process#readme), including:
+  - [Finding things to work on](https://github.com/tektoncd/community/tree/main/process#finding-something-to-work-on)
+  - [Proposing new features](https://github.com/tektoncd/community/tree/main/process#proposing-features)
+  - [Performing code reviews](https://github.com/tektoncd/community/tree/main/process#reviews)
+  - [Becoming a code owner](https://github.com/tektoncd/community/blob/main/OWNERS)
```

Comment:
thank you @pavanstarmanwar for fixing broken links 👍 

can you please replace the title and link with:

```
- [Becoming a community member and maintainer](https://github.com/tektoncd/community/blob/main/process/contributor-ladder.md)
```

@lbernick thoughts?

This `owners' link to showing a list of folks on Governing Board but the intention here it to showcase the Tekton contributor ladder to a new contributor.

Reply:
I'd actually replace this line with
```
- [Moving up the contributor ladder](https://github.com/tektoncd/community/tree/main/process/contributor-ladder.md)

Reply:
I just made the above change and pushed a commit. :)
Let me know which one and I will update the same

Reply:
This looks fine to me!